pkg/costmodel/costmodel.go

```diff
@@ -85,6 +85,7 @@ type CostData struct {
 	GPUReq          []*util.Vector               `json:"gpureq,omitempty"`
 	PVCData         []*PersistentVolumeClaimData `json:"pvcData,omitempty"`
 	NetworkData     []*util.Vector               `json:"network,omitempty"`
+	Annotations     map[string]string            `json:"Annotations,omitempty"`
```

Comment:
`json:"annotations,omitempty"` camel case for json properties

Reply:
updated